CHANGELOG.md

```diff
@@ -1,5 +1,29 @@
 # Changelog
 
+## [0.97.0](https://github.com/Snowflake-Labs/terraform-provider-snowflake/compare/v0.96.0...v0.97.0) (2024-10-10)
```

Comment:
just wondering where we changed this version to v0.97.0 OR we are going to do it somewhere?

Reply:
it seems like later we will get the new v0.97.0 version
https://github.com/Snowflake-Labs/terraform-provider-snowflake/releases